aspnetcore/fundamentals/host/web-host.md

```diff
@@ -2,21 +2,22 @@
 title: ASP.NET Core Web Host
 author: guardrex
 description: Learn about the web host in ASP.NET Core, which is responsible for app startup and lifetime management.
+monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 10/18/2018
+ms.date: 11/05/2018
 uid: fundamentals/host/web-host
 ---
 # ASP.NET Core Web Host
 
 By [Luke Latham](https://github.com/guardrex)
 
+For the 1.1 version of this topic, download [ASP.NET Core Web Host (version 1.1, PDF)](https://webpifeed.blob.core.windows.net/webpifeed/Partners/Web-Host_1.1.pdf).
```

Comment:
Yes ... with 1.x supported until June, my understanding is that the 1.1 coverage in the PDF is sufficient to tide us over to EOL. Since 2.0 is unsupported, my understanding is that it can be dropped now.

Although the 1.1 topic coverage is provided in the PDF, it remains covered in the API browser for those who require it at least until June, possibly even after June.

We'll get the approval from APEX before we merge this just to make sure.

cc: @wadepickett @Rick-Anderson